spec/api/v1/users_controller_spec.rb

```diff
@@ -3,53 +3,56 @@
 describe Api::V1::UsersController do
   describe 'GET /api/users' do
     it 'returns all users' do
-      10.times do
-        create(:user)
+      3.times do |i|
+        create(:user, name: "name_#{i}", email: "email_#{i}@example.com")
       end
-      create(:user, name: 'Janusz', email: 'fake.mail@example.com')
 
       get '/api/v1/users'
 
       expect(response.status).to eq(200)
-      expect(JSON.parse(response.body).count).to eq 11
-      expect(JSON.parse(response.body).last['name']).to eq 'Janusz'
-      expect(JSON.parse(response.body).last['email']).to eq 'fake.mail@example.com'
+      expect(JSON.parse(response.body).count).to eq 3
```

Comment:
I do not see benefits. With module I have to use sth like:
```JsonResponse.json_response(response)``` (which looks weird for me)
instead of
```JSON.parse(response.body)```